tests/test_misc.py

```diff
@@ -100,3 +101,62 @@ def test_to_list_of_strings(value, expected):
 )
 def test_to_list_without_none(value, expected):
     assert to_list_without_none(value) == expected
+
+
+inp1 = """# record 1
+dn: cn=1
+cn: sssd
+config: 22
+
+# record 2
+dn: cn=2
+version: 22
+"""
+
+out1 = {
+    "cn=1": {"dn": ["cn=1"], "cn": ["sssd"], "config": ["22"]},
+    "cn=2": {"dn": ["cn=2"], "version": ["22"]},
+}
+
+inp2 = """# record 1
+dn: cn=1
+cn: 1
+debug: prvni
+
+# record 2
+debug: prvni
+dn: cn=2
+cn: 2
+debug: druhy
+"""
+
+out2 = {
+    "cn=1": {"dn": ["cn=1"], "cn": ["1"], "debug": ["prvni"]},
+    "cn=2": {"dn": ["cn=2"], "cn": ["2"], "debug": ["prvni", "druhy"]},
+}
+
+inp3 = """# record 1
+dn: cn=sssd
+cn: sssd
+cn: sssd
+
+# returned 1 records
+# 1 entries
+# 0 referrals
+"""
+
+out3 = {
+    "cn=sssd": {"dn": ["cn=sssd"], "cn": ["sssd", "sssd"]},
+}
+
+
+@pytest.mark.parametrize(
+    "value,expected",
+    [
+        (inp1, out1),
```

Comment:
Expand the variables here, please. You can use nice indentation, then dedent it with textwrap.dedent().

Using vague names like inp1, out1 does not work very well, since you can have more parametrized tests.